audit/src/main/scala/tech/beshu/ror/audit/AuditFieldValue.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.audit
+
+private[ror] sealed trait AuditFieldValue
```

Comment:
`private[ror]`
Is it enough?
When we extend the sealed trait values set in the future, are we sure that users won't have to recompile their custom serializers?

Reply:
The changes are in a single commit - https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/pull/1140/commits/2bb638afa267d936bff319f3910947c47f48ed9d